### ISSUE TITLE
Love the game!

### ISSUE THREAD

**Issue by shamoons** [ntekal/clickGame#1]
* Consider your folder structure. You have everything in the main root. You should instead put images into an `images` folder, CSS into a `css` folder and JavaScript into a `js` folder
* Give some indication that the game has started
* We never want to use `onclick` in HTML. Instead add event listeners in JavaScript
* Focus on your indentation. Consider https://github.com/ntekal/clickGame/blob/master/clickGame.html#L48. The `div` should close at the same indentation level that it opens at

**Comment by ntekal** [comment]
@shamoons Yeah I realized after submitting when I spoke with Matt and Evan that we should never be using onclick so I'm going to re-do my click game so that the folder structure is more organized, my indentation is cleaner, and I'm going to improve the features of my game as well -- I'll comment again once it is complete!
